recipes/mgltools/meta.yaml

```diff
@@ -0,0 +1,25 @@
+about:
+    home: http://mgltools.scripps.edu/
+    license_file: LICENSES
+    summary: MGLTools is an program for visualization and analisys of molecular structures.
+package:
+    name: mgltools
+    version: 1.5.6
+build:
+  number: 0
+  skip: False
+source:
+    fn:  mgltools_x86_64Linux2_1.5.6.tar.gz                                                                             # [linux]
+    url: http://mgltools.scripps.edu/downloads/downloads/tars/releases/REL1.5.6/mgltools_x86_64Linux2_1.5.6.tar.gz      # [linux]
```

Comment:
Are these source packages, or binary distributions? If you download two separate binaries you should use the template syntax to specify when to use which URL. Eg. # [osx].

Compiling from source in encouraged if possible :)